.circleci/config.yml

```diff
@@ -28,8 +28,34 @@ jobs:
       - run:
           name: Run tests
           command: node common/scripts/install-run-rush.js test --verbose
+  update_npm_dependencies:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - run:
+          name: Install needed npm packages
+          command: npm install npm-check-updates @octokit/rest --no-save
```

Comment:
While this is not recommended by rush (it introduces [phantom dependencies](https://rushjs.io/pages/advanced/phantom_deps/)), it should be safe enough in this flow.

Reply:
I've had the same though while writing this